src/Enums/AvailableNewsletter.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Enums;
+
+enum AvailableNewsletter: string
+{
+    case NEWS = "news";
+    case MEETUPS = "meetups";
```

Comment:
I think cases should be CamelCase. `News` and `Meetups`

As it is in PHP Docs:
https://www.php.net/manual/en/language.enumerations.backed.php